sopel/db.py

```diff
@@ -481,7 +493,7 @@ def unalias_nick(self, alias):
             To *add* an alias for a nick, use :meth:`alias_nick`.
```

Comment:
…which got type hints added to its definition, but `unalias_nick` didn't.

Neither did `forget_nick_group` and several other related functions below. I don't understand why some touched functions have hinting added and some don't.

Reply:
Yes that's not very thorough: I tend to type hint what I need to understand or modify, and not everything. Given how much I've changed in SopelDB, I'll go through every methods to fix that.